src/controllers/userController.ts

```diff
@@ -56,6 +59,52 @@ class UserController {
             next(e);
         }
     }
+
+    public async addUserProfileInfo(req, res, next): Promise<UserProfileInfo> {
+        try {
+            const errors = validationResult(req);
+            if (!errors.isEmpty()) {
+                return next(
+                    ApiError.BadRequest('Validation error', errors.array()),
+                );
+            }
+
+            const {
+                name,
+                surname,
+                email,
+                skype,
+                ageExperience,
+                department,
+                summary,
+            } = req.body;
+
+            const userProfileData = await profileService.addUserProfileInfo(
+                name,
+                surname,
+                email,
+                skype,
+                ageExperience,
+                department,
+                summary,
+            );
+
+            return res.json(userProfileData);
+        } catch (e) {
+            next(e);
+        }
+    }
+
+    public async getUserProfileInfo(req, res, next): Promise<UserProfileInfo> {
+        try {
+            const id = req.user.id;
```

Comment:
after u add type for req u'll get an error that req doesn't have a user
find how to extend global req type